platforms/quorum/charts/quorum-tessera-node/templates/statefulset.yaml

```diff
@@ -0,0 +1,321 @@
+  updateStrategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: {{ include "goquorum-tessera-node.fullname" . }}
+      app.kubernetes.io/name: tessera-statefulset
+      app.kubernetes.io/component: goquorum
+      app.kubernetes.io/part-of: {{ include "goquorum-tessera-node.fullname" . }}
+      app.kubernetes.io/managed-by: {{ .Release.Service }}
+      app.kubernetes.io/namespace: {{ .Release.Namespace }}
+      app.kubernetes.io/release: {{ .Release.Name }}
+  serviceName: {{ include "goquorum-tessera-node.fullname" . }}
+  volumeClaimTemplates:
+    - metadata:
+        name: data
+        labels:
+          {{- if $.Values.labels }}
+          {{- range $key, $value := $.Values.labels.pvc }}
+          {{- range $k, $v := $value }}
+          {{ $k }}: {{ $v | quote }}
+          {{- end }}
+          {{- end }}
+          {{- end }}
+      spec:
+        storageClassName: storage-{{ .Release.Name }}
+        accessModes: [ "ReadWriteOnce" ]
+        resources:
+          requests:
+            storage: {{ .Values.storage.size }}
+    - metadata:
+        name: data-mysql
+        labels:
+          {{- if $.Values.labels }}
+          {{- range $key, $value := $.Values.labels.pvc }}
+          {{- range $k, $v := $value }}
+          {{ $k }}: {{ $v | quote }}
+          {{- end }}
+          {{- end }}
+          {{- end }}
+      spec:
+        storageClassName: storage-{{ .Release.Name }}
+        accessModes: [ "ReadWriteOnce" ]
+        resources:
+          requests:
+            storage: {{ .Values.storage.dbSize }}
+  template:
+    metadata:
+      labels:
+        app: {{ include "goquorum-tessera-node.fullname" . }}
+        app.kubernetes.io/name: tessera-statefulset
+        app.kubernetes.io/component: goquorum
+        app.kubernetes.io/part-of: {{ include "goquorum-tessera-node.fullname" . }}
+        app.kubernetes.io/managed-by: {{ .Release.Service }}
+        app.kubernetes.io/namespace: {{ .Release.Namespace }}
+        app.kubernetes.io/release: {{ .Release.Name }}
+        {{- if $.Values.labels }}
+        {{- range $key, $value := $.Values.labels.deployment }}
+        {{- range $k, $v := $value }}
+        {{ $k }}: {{ $v | quote }}
+        {{- end }}
+        {{- end }}
+        {{- end }}
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/port: {{ .Values.tessera.metrics.port | quote }}
+        prometheus.io/path: "/metrics"
+    spec:
+      serviceAccountName: {{ .Values.global.serviceAccountName }}
+      hostname: {{ include "goquorum-tessera-node.fullname" . }}.{{ .Release.Namespace }}
+      imagePullSecrets:
+      {{- if .Values.image.pullSecret }}
+        - name: {{ .Values.image.pullSecret }}
+      {{- end }}
+      initContainers:
+      - name: certificates-init
+        image: {{ .Values.image.alpineutils.repository }}:{{ .Values.image.alpineutils.tag }}
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: VAULT_ADDR
```

Comment:
vault.type check is missing in this file

Reply:
Fixed.